src/domain/use-cases/email-service.ts

```diff
@@ -0,0 +1,22 @@
+export interface EmailService {
+  send: (options: EmailService.Options) => Promise<EmailService.Result>
+}
+
+export namespace EmailService {
+  export interface Transport {
```

Comment:
Transport == Params ?

Reply:
I don't think so. Maybe you could change "EmailService.Options" to "EmailService.Params" since that is what is being used in the "send" method. What do you think?